docs/src/content/docs/ko/resources/plugins.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: 플러그인 및 통합
+description: Starlight를 확장하는 플러그인 및 통합과 같은 커뮤니티 도구를 찾아보세요!
+sidebar:
+  order: 1
+---
+
+:::tip[직접 추가하세요!]
+Starlight용 플러그인이나 도구를 구축하셨나요?
+이 페이지에 링크를 추가하여 PR을 열어보세요!
+:::
+
+## 플러그인
+
+[플러그인](/ko/reference/plugins/)은 Starlight 구성, UI 및 동작을 사용자 정의하는 동시에 쉽게 공유하고 재사용할 수 있습니다.
+Starlight 팀이 지원하는 공식 플러그인과 Starlight 사용자가 관리하는 커뮤니티 플러그인으로 사이트를 확장하세요.
+
+### 공식 플러그인
+
+<CardGrid>
+	<LinkCard
+		href="/guides/site-search/#algolia-docsearch"
```

Comment:
```suggestion
		href="/ko/guides/site-search/#algolia-docsearch"
```

Just a small detail, this can be changed to the version in the proper locale.

Reply:
Thanks for the good point!

I forgot to edit the URL 👍